### ISSUE TITLE
Accessibility: Location map is entirely inaccesible for screen-reader users

### ISSUE THREAD

**Issue by Murcielago7384** [mysociety/fixmystreet#6139]
**Describe the bug**
While generating a report, and after entering a postcode, the resulting map is not accessible to those using a screen-reader. It is possible to read the instructions, I.E. regarding using the + and - keys, but there is no way to actually interact and choose a location.

**To Reproduce**
To test this, you will need a screen-reader. NVDA is a free and open source screen-reader for Windows, available from NV Access.
Open any Fix My Street branch (".co.uk" in my case) in any browser, and enter a postcode in the search box.
When the resulting map is displayed, try to interact with the page with either the tab key, or the arrow keys.

**Expected behavior**
Maps are notoriously difficult to navigate for screen-reader users. Ideally there should be an option under the map that would allow you to "manually enter an address instead", which could either search based on the previously entered postcode, or have a manual entry similar to what you would find on a contact form that asks for an address.

**Desktop (please complete the following information):**
Windows 11, with Firefox Browser 155.0.1. NVDA 
version 2026.2.

**Additional Context**
As I mentioned above, NVDA is a free and open source screen-reader available for Windows. I am aware that there is also a FMS application for Android. If screen-reader-based accessibility tests was something the devs were interested in, I would be keen to help with that as well.

**Comment by zarino** [comment]
Hi @Murcielago7384. Thanks for reporting the issue.

There’s a “Can’t use the map? Skip this step” link in the page (right after the “Click map to report a problem” H1). It is absolutely positioned off screen, but becomes visible when you tab to it. Did you spot that, or do we need to improve that?

If you press that “Skip this step” button, FixMyStreet will use whatever location you initially entered into the homepage search box as the location to be sent along with your report. Our thinking here is that, hopefully, the combination of that general location, plus any detail you put into the issue description, will be enough for the council to process the report, without having to inconvenience screen reader users with an extra location input.

**Comment by Murcielago7384** [comment]
Hi @zarino
Sorry about the delay in replying here. I was aware of the "skip map" link, but I didn't know the original postcode entered is sent along with the report. I was under the impression that skipping the map meant you were entirely reliant on the description.
That is very good to know, thanks. And I appreciate your quick reply to my issue! :)
